### PR TITLE
rearranges getting-started.udon so it's easier to follow

### PR DESCRIPTION
**Pull request by rmariani**
People reported that they found the getting-started page hard to follow. This makes it more straightforwardly a map to our getting-started guides.

### CODE REVIEW

**Comment by rmariani** [comment]
@msutherl curious about your thoughts

**Comment by msutherl** [comment]
@rmariani left some comments. Seems like an iteration in a positive direction overall.

**Comment by rmariani** [comment]
Alright, too your suggestions into account @msutherl

File: getting-started.udon [hunk 1]
```diff
@@ -7,36 +7,31 @@
 
 # Getting Started
 
-Pages in this section are guides to getting you started with Urbit.
+Welcome to Urbit, a personal server that makes you the owner of your digital life. Urbit isn't ready for serious use yet, but it definitely works. Follow this step-by-step guide to get an Azimuth point, install Urbit, and join the community.
```
Comment (msutherl) [inline getting-started.udon]: This feels a little off to me still.

I wonder if the first sentence should stay vague. "Makes you the owner of your digital life" sets up the expectation that there's actually software in existence to do that. Right now Urbit doesn't actually provide that experience. How about just "Welcome to Urbit, a personal server."

"Urbit isn't ready for serious use yet, but it definitely works" also worries me – I'm not sure if we can say "definitely works" and mean it. "Urbit isn't ready for serious use yet, but if you're curious you can try it out. Follow this step-by-step guide to get an Azimuth point, install Urbit, and talk to other users on the network."

File: getting-started.udon [hunk 2]
```diff
@@ -7,36 +7,31 @@
...
-### [Booting a Ship](booting-a-ship)
-
-This guide will walk you through the process of getting your ship operational and on the Arvo network.
-
-### [Creating a Development Ship](creating-a-development-ship)
-
-If you just want to do development work with Arvo, and don't need to be on the Urbit network, this section is for you.
+- Arvo is the name of the Urbit operating system.
+- An instance of the Arvo operating system is called a _ship_.
+- The network linking ships on this network is called "the Arvo network."
```
Comment (msutherl) [inline getting-started.udon]: This is tautological :) Maybe "Ships can talk to each other over 'the Arvo network.'"

File: getting-started.udon [hunk 3]
```diff
@@ -7,36 +7,31 @@
...
+- Azimuth is the name of the Urbit identity layer.
```
Comment (msutherl) [inline getting-started.udon]: Maybe "Azimuth is an Ethereum-based PKI used to give each ship a unique identity on the Arvo network"

File: getting-started.udon (hunk 1 above)
Comment (rmariani) [reply getting-started.udon]: How about "but it works as alpha-stage software."

File: getting-started.udon [hunk 4]
```diff
@@ -7,36 +7,31 @@
...
+- If you want to get on the Arvo network, an Azimuth-based identity called a _point_.
```
Comment (msutherl) [inline getting-started.udon]: "An Azimuth identity is called a 'point.' You'll need a point to use the Arvo network.

File: getting-started.udon [hunk 5]
```diff
@@ -7,36 +7,31 @@
...
+- Urbit is the project as the whole, which includes all of these elements and more.
```
Comment (msutherl) [inline getting-started.udon]: Typo. Maybe "'Urbit' refers to both the `urbit` program that runs Arvo and Arvo's interpreter, Vere, and the entire project including both Arvo and Azimuth."

File: getting-started.udon (hunk 1 above)
Comment (msutherl) [reply getting-started.udon]: We are [pre-alpha](https://en.wikipedia.org/wiki/Software_release_life_cycle#Pre-alpha), but I wouldn't use this terminology at all.

File: getting-started.udon (hunk 5 above)
Comment (rmariani) [reply getting-started.udon]: This seems like an inordinate info-dump

File: getting-started.udon (hunk 1 above)
Comment (rmariani) [reply getting-started.udon]: I think this is a matter of debate, and "alpha" properly carries the connotation of "rudimentary"

File: getting-started.udon (hunk 5 above)
Comment (msutherl) [reply getting-started.udon]: "Urbit refers to the project as a whole including Arvo, Azimuth, and other system components."

Comment (msutherl) [reply getting-started.udon]: Mainly "elements" is vague and "and more!" I'd avoid

File: getting-started.udon (hunk 1 above)
Comment (rmariani) [reply getting-started.udon]: Alright, I am just going to drop the terminology

**Comment by msutherl** [comment]
@rmariani we'll have to create a new PR for this after `zola-port` is merged
